store/cafs/test/index.ts

```diff
@@ -43,6 +43,16 @@ describe('cafs', () => {
     expect(await fs.readFile(filePath, 'utf8')).toBe('foo\n')
     expect(await manifest.promise).toEqual(undefined)
   })
+
+  it('ignores broken symlinks when traversing subdirectories', async () => {
+    const storeDir = tempy.directory()
+    const srcDir = path.join(__dirname, 'fixtures/broken-symlink')
```

Comment:
How is this a broken symlink? This is just a file that doesn't exist.

Reply:
It's a symlink that points nowhere, hence broken.